iroha-lib/model/Tx.hpp

```diff
@@ -0,0 +1,73 @@
+#ifndef TX_HPP
+#define TX_HPP
+
+#include "LibsAndClassDeclarations.h"
+
+#include "transaction.pb.h"
+#include <boost/bimap.hpp>
+#include "crypto/keypair.hpp"
+#include "generators/CommandGenerator.hpp"
+
+
+namespace iroha_lib
+{
+
+class Tx
+{
+    iroha::keypair_t keypair_;
+    iroha::protocol::Transaction protobuf_transaction_;
+    CommandGenerator cmd_generator_;
+
+public:
+    Tx(const std::string& account_id,
+       const iroha::keypair_t& keypair,
+       uint64_t created_time = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(),
+       uint32_t quorum = 1);
```

Comment:
Consider marking it as `explicit`, and inlining the definition and using initialiser lists.

Reply:
Changed.